api/v1beta1/types.go

```diff
@@ -114,6 +114,21 @@ type NetworkSpec struct {
 	LoadBalancerBackendPort *int32 `json:"loadBalancerBackendPort,omitempty"`
 }
 
+// LoadBalancerSpec contains configuration for one or more LoadBalancers.
+type LoadBalancerSpec struct {
+
+	// APIServerInstanceGroupTagOverride overrides the default setting for the
+	// tag used when creating the API Server Instance Group.
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinLength=1
```

Comment:
Honestly not sure how an optional field and a minimum length work together in these generated validations.

Reply:
Removed the MinLength check to avoid any ambiguity.